knowledge-base/index.html

```diff
@@ -0,0 +1,44 @@
+---
+layout: wide
+permalink: /knowledge-base/
```

Comment:
@apburnes I haven't included pagination for this yet, and with a small(ish) number of articles that won't be a problem. But at some point we'll want to add it. I ran into some issues adding pagination for a new collection. Any thoughts or suggestions welcomed.

Reply:
Just realized this comment existed.  I will tinker a bit.